README.md

```diff
@@ -98,11 +98,13 @@ Wechaty already held lots of talk and got a lot of blogs in the past 4 years, he
 
 import { WechatyBuilder } from 'wechaty'
 
-WechatyBuilder.build()  // get a Wechaty instance
+const wechaty = WechatyBuilder.build({}) // get a Wechaty instance
```

Comment:
Remove the unnecessary `{}` to make the code clear

Reply:
fixed